<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTableWidget" name="TableLyric"/>
    </item>
    <item row="0" column="1">
     <widget class="QTableWidget" name="TableList">
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>歌曲列表</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="2" column="1">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="2">
       <widget class="QPushButton" name="ButtonNext">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="iconsource.qrc">
          <normaloff>:/Last.ico</normaloff>:/Last.ico</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="ButtonLast">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="iconsource.qrc">
          <normaloff>:/previous.ico</normaloff>:/previous.ico</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="ButtonPlay">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="iconsource.qrc">
          <normaloff>:/play.ico</normaloff>:/play.ico</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" rowspan="2">
     <widget class="QLabel" name="CurrentMusic">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QSlider" name="VedioSlider">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1041</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="iconsource.qrc"/>
 </resources>
 <connections/>
</ui>
